core/cli/package.json

```diff
@@ -41,15 +41,15 @@
     "zod": "^3.22.4"
   },
   "dependencies": {
-    "@dotcom-tool-kit/base": "^3.4.1",
-    "@dotcom-tool-kit/config": "^1.0.0",
-    "@dotcom-tool-kit/conflict": "^1.0.0",
-    "@dotcom-tool-kit/error": "^3.2.0",
-    "@dotcom-tool-kit/logger": "^3.4.0",
-    "@dotcom-tool-kit/options": "^3.2.0",
-    "@dotcom-tool-kit/plugin": "^1.0.0",
-    "@dotcom-tool-kit/validated": "^1.0.0",
-    "@dotcom-tool-kit/wait-for-ok": "^3.2.0",
+    "@dotcom-tool-kit/base": "4.0.0-beta.0",
+    "@dotcom-tool-kit/config": "2.0.0-beta.0",
+    "@dotcom-tool-kit/conflict": "2.0.0-beta.0",
+    "@dotcom-tool-kit/error": "4.0.0-beta.0",
+    "@dotcom-tool-kit/logger": "4.0.0-beta.0",
+    "@dotcom-tool-kit/options": "4.0.0-beta.0",
+    "@dotcom-tool-kit/plugin": "2.0.0-beta.0",
+    "@dotcom-tool-kit/validated": "2.0.0-beta.0",
+    "@dotcom-tool-kit/wait-for-ok": "4.0.0-beta.0",
```

Comment:
**question**: was it intentional to pin the dependencies? caret ranges still work for prereleases